editor.planx.uk/src/pages/Teams.tsx

```diff
@@ -72,9 +77,49 @@ const Teams: React.FC<Props> = ({ teams, teamTheme }) => {
     });
   return (
     <Container maxWidth="formWrap">
-      <Typography variant="h2" component="h1" mb={4}>
-        Select a team
-      </Typography>
+      <Box
+        pb={1}
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "space-between",
+          alignItems: "center",
+          marginBottom: "40px",
+        }}
+      >
+        <Typography variant="h2" component="h1">
+          Select a team
+        </Typography>
+        {isPlatformAdmin ? (
+          <AddButton
+            onClick={async () => {
+              const newTeamName = prompt("Team name");
+
+              if (newTeamName) {
+                const newSlug = slugify(newTeamName);
+                const teamNameDuplicate = teams.find(
+                  (team) => team.slug === newSlug,
+                );
+                if (teamNameDuplicate === undefined) {
+                  await useStore
+                    .getState()
```

Comment:
nit: We could simplify this by extracting `create()` (or `createTeam()` 😉) at the top of the component.

```ts
const [canUserEditTeam, createTeam] = useStore((state) => [state.canUserEditTeam, state.createTeam])
```